packages/wrangler/src/__tests__/completions.test.ts

```diff
@@ -0,0 +1,258 @@
+import { execSync } from "child_process";
+import { describe, test } from "vitest";
+import { mockConsoleMethods } from "./helpers/mock-console";
+import { runInTempDir } from "./helpers/run-in-tmp";
+import { runWrangler } from "./helpers/run-wrangler";
+
+function shellAvailable(shell: string): boolean {
+	try {
+		execSync(`which ${shell}`, { stdio: "ignore" });
+		return true;
+	} catch {
+		return false;
+	}
+}
+
+describe("wrangler", () => {
+	describe("completions", () => {
+		const std = mockConsoleMethods();
+		runInTempDir({ homedir: "./home" });
```

Comment:
`"./home"` is already the default.
```suggestion
		runInTempDir();
```